src/afl-fuzz-stats.c

```diff
@@ -283,6 +284,28 @@ void write_stats_file(afl_state_t *afl, u32 t_bytes, double bitmap_cvg,
 
   }
 
+  if (afl->schedule >= FAST && afl->schedule <= RARE) {
+
+    u64 singletons = 0;
+    for (u32 i = 0; i < N_FUZZ_SIZE; ++i) {
+
+      if (afl->n_fuzz_stats[i] == 1) { ++singletons; }
+
+    }
+
+    // Compute the Good-Turing estimates
+    if (afl->fsrv.total_execs > 0) {
+
+        u64 gt = (singletons /afl->fsrv.total_execs);
```

Comment:
Here you redefine gt in a local scope. As soon as you leave the if statement, the value is forgotten.
Instead, remove the variable type (u64) to let this value be assigned to src/afl-fuzz-stats.c#L254.